applications/plugins/SofaCUDA/sofa/gpu/cuda/CudaBaseVector.cu

```diff
@@ -19,8 +19,8 @@
 *                                                                             *
 * Contact information: contact@sofa-framework.org                             *
 ******************************************************************************/
-#include "CudaCommon.h"
-#include "CudaMath.h"
+#include <sofa/gpu/cuda/CudaCommon.h>
```

Comment:
Wouldn't be worth it to start with those guys ?
SofaCUDA/type/CudaCommon.h
or
SofaCUDA/helper/CudaCommon

Reply:
It is also in my plan to move those files. I started by the components because it is the easiest IMO. And also because I did not anticipate the newest structure (the one with `SofaCUDA/src/SofaCUDA`.